docs/wiki/deployment/yara.md

```diff
@@ -0,0 +1,138 @@
+| /Users/wxs/tmp/foo | tmp      | 1430078285 | CREATED | 33859499       | d41d8cd98f00b204e9800998ecf8427e | da39a3ee5e6b4b0d3255bfef95601890afd80709 | e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855 |
++--------------------+----------+------------+---------+----------------+----------------------------------+------------------------------------------+------------------------------------------------------------------+
+osquery> select * from yara_events;
++--------------------+----------+------------+---------+----------------+-------------+-------+
+| target_path        | category | time       | action  | transaction_id | matches     | count |
++--------------------+----------+------------+---------+----------------+-------------+-------+
+| /Users/wxs/tmp/foo | tmp      | 1430078285 | CREATED | 33859499       | always_true | 1     |
++--------------------+----------+------------+---------+----------------+-------------+-------+
+osquery> 
+```
+
+The **file_changes** table recorded that a file named */Users/wxs/tmp/foo* was created with the corresponding hashes and a timestamp.
+
+The **yara_events** table recorded that 1 matching rule (*always_true*) was found when the file was created. In this example every file will always have at least one match because I am using a rule which always evaluates to true. In this next example I'll issue the same command to create a file in a monitored directory but have removed the *always_true* rule from my signature files.
+
+```
+osquery> select * from yara_events;
++--------------------+----------+------------+---------+----------------+-------------+-------+
+| target_path        | category | time       | action  | transaction_id | matches     | count |
++--------------------+----------+------------+---------+----------------+-------------+-------+
+| /Users/wxs/tmp/foo | tmp      | 1430078285 | CREATED | 33859499       | always_true | 1     |
+| /Users/wxs/tmp/foo | tmp      | 1430078524 | CREATED | 33860795       |             | 0     |
++--------------------+----------+------------+---------+----------------+-------------+-------+
+osquery> 
+```
+
+As you can see, even though no matches were found an row is still created and stored.
+
+# yara table
+
+The **yara** table is used for on-demand scanning. With this table you can arbitrarily YARA scan any available file on the filesystem with any available signature files or signature group from the configuration. In order to scan the table must be given a constraint which says where to scan and what to scan with.
+
+In order to determine where to scan the table accepts either a *path* or a *pattern* constraint. The *path* constraint must be a full path to a single file. There is no expansion or recursion with this constraint. The *pattern* constraint follows the same [wildcard rules](http://osquery.readthedocs.org/en/latest/development/wildcard-rules/index.html) mentioned before.
```

Comment:
Do you think it's worth us adding stand-alone documentation somewhere on the logic behind the whole `pattern = "some/pattern"` thing? People in chat have called it out for being a bit confusing.